iam/certificate-acl/Dockerfile

```diff
@@ -36,14 +36,10 @@ RUN apk --no-cache add \
     redis \
     libsasl \
     lz4-libs \
-    tini 
+    tini
 
 COPY --from=build /opt/certificate-acl /opt/certificate-acl
 
 ENTRYPOINT ["/sbin/tini", "--"]
 
-CMD ["npm", "run", "certificate-acl"]
-
-HEALTHCHECK --start-period=5s --interval=30s --timeout=5s --retries=3 \
-      CMD HC=$(wget -q -O - "http://localhost:9000/health") \
-        && echo $HC && echo $HC | grep -q 'SERVER_IS_READY' 
\ No newline at end of file
+CMD ["npm", "run", "certificate-acl"]
```

Comment:
Please add `HEALTHCHECK --start-period=2m --interval=30s --timeout=10s --retries=3 \
    CMD curl -f http://localhost:9000/health || exit 1 `.  I think it's a good idea, since it can be overwritten when using docker-compose.